requirements.txt

```diff
@@ -3,6 +3,6 @@ Jinja2
 lightkube
 lightkube-models
 ops >= 2.2.0
-pydantic ~= 2.5.3
+pydantic >= 2.5.3
```

Comment:
I would suggest to keep this as it was, no need to change the charm's package version.

Reply:
Ok.

I did not know what was the relationship between the `requirements.txt` dependency constraint and the ones found in the  `PYDEPS` constants. Now I know 👍🏻